tests/test_export_controller.py

```diff
@@ -91,6 +91,30 @@ def test_export_color_images(tmp_dir):
         assert os.path.exists(plots_dir / image)
 
 
+def test_export_color_images_interlace(tmp_dir):
```

Comment:
Maybe better `test_export_color_images_plane(...)` ?

Reply:
Sure, why not.